typing_extensions/src_py3/test_typing_extensions.py

```diff
@@ -471,7 +474,10 @@ def test_counter_instantiation(self):
         class C(typing_extensions.Counter[T]): ...
         if TYPING_3_5_3:
```

Comment:
It's kind of unfortunate that these variables are named in such a way that it's not clear that they imply "or all higher versions".

Reply:
I agree, but I would say it is too late to change this.